orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/VectorOfTrust.java

```diff
@@ -70,20 +70,9 @@ public static VectorOfTrust parseFromAuthRequestAttribute(List<String> vtr) {
             throw new IllegalArgumentException("Invalid VTR attribute", e);
         }
         List<VectorOfTrust> vtrList = parseVtrSet(vtrJsonArray);
-        var orderedVtrList = orderVtrList(vtrList);
-        LOG.info("VTR has been processed at vectorOfTrust: [{}]", stringifyVtrList(orderedVtrList));
-
-        // Background experiment to ensure no breaking changes are made with removing the existing
-        // parsing
-        var minimumVtr = orderedVtrList.get(0);
-        var existingMinimumVtr = existingGetMinimumVot(vtrList);
-
-        if (minimumVtr.equals(existingMinimumVtr)) {
-            return minimumVtr;
-        } else {
-            LOG.warn("Methods are not equivalent");
-            return existingMinimumVtr;
-        }
```

Comment:
As discussed, we're going to let this soak for a couple of days before merging this change